client/app/bundles/course/assessment/submission/components/ScribingView/ToolDropdown.jsx

```diff
@@ -40,44 +42,89 @@ const style = {
     fontSize: '12px',
     padding: '10px 0px 10px 0px',
   },
+  disabled: {
+    cursor: 'not-allowed',
+    pointerEvents: 'none',
+    color: '#c0c0c0',
+  },
 };
 
 export default class ToolDropdown extends Component {
   renderIcon() {
-    const { iconClassname, currentTool, toolType, iconComponent } = this.props;
+    const { disabled, iconClassname, currentTool, toolType, iconComponent } = this.props;
+    let iconStyle;
+    if (disabled) {
+      iconStyle = style.disabled;
+    } else if (currentTool === toolType) {
+      iconStyle = { color: blue500 };
```

Comment:
To improve readability and consistency, perhaps can do `iconStyle = style.selected`?